scripts/run-functional-tests.sh

```diff
@@ -20,8 +23,8 @@ docker run \
   -v $(pwd)/node_modules/testcafe-react-selectors:/opt/testcafe/node_modules/testcafe-react-selectors \
   -e NODE_PATH=/opt/testcafe/node_modules:/opt \
   -e BOOKITURI=$BOOKITURI \
-  -e BOOKITUSER=$BOOKITUSER \
-  -e BOOKITPASSWD=$BOOKITPASSWD \
+  -e BOOKITUSER=$(aws ssm get-parameters --names BUILDIT_REGULAR_USER_NAME --with-decryption --output text | cut -f 4) \
```

Comment:
What does "regular" mean here.  Is there an "irregular" user name somewhere else?   But seriously, what makes it "regular"?  Is there a better name?